syntax = "proto3";

package PaperSoccer;

enum Direction {
  Top = 0;
  TopRight = 1;
  Right = 2;
  BottomRight = 3;
  Bottom = 4;
  BottomLeft = 5;
  Left = 6;
  TopLeft = 7;
};
